modules/multivcfanalyzer/main.nf

```diff
@@ -0,0 +1,72 @@
+process MULTIVCFANALYZER {
+    tag '$fasta'
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::multivcfanalyzer=0.85.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/multivcfanalyzer:0.85.2--hdfd78af_1':
+        'quay.io/biocontainers/multivcfanalyzer:0.85.2--hdfd78af_1' }"
+
+    input:
+    path vcfs
+    path fasta
+    path snpeff_results
+    path gff
+    val allele_freqs
+    val genotype_quality
+    val coverage
+    val homozygous_freq
+    val heterozygous_freq
+    path gff_exclude
+
+
+    output:
+    path('fullAlignment.fasta.gz')                       , emit: full_alignment
+    path('info.txt')                                     , emit: info_txt
+    path('snpAlignment.fasta.gz')                        , emit: snp_alignment
+    path('snpAlignmentIncludingRefGenome.fasta.gz')      , emit: snp_genome_alignment
+    path('snpStatistics.tsv')                            , emit: snpstatistics
+    path('snpTable.tsv')                                 , emit: snptable
+    path('snpTableForSnpEff.tsv')                        , emit: snptable_snpeff
+    path('snpTableWithUncertaintyCalls.tsv')             , emit: snptable_uncertainty
+    path('structureGenotypes.tsv')                       , emit: structure_genotypes
+    path('structureGenotypes_noMissingData-Columns.tsv') , emit: structure_genotypes_nomissing
+    path('MultiVCFAnalyzer.json')                        , emit: json
+    path "versions.yml"                                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
```

Comment:
Why not removing that if you don't use it?
Is it necessary for linting?

Reply:
Consistency.

I assumed it's better to call `args2` to indicate that it's for the second part of the module (gzipping). I could remove the args (1) line entirely. I don't want to confuse a future reviewer of the module wondering where args1 was.

I also worry someone would try adding it to `modules.conf` (even if it does nothing) and it could cause errors? Although maybe not thinking about it :thinking:  

Don't mind ultaimtely - what do you think?